editor/editor_themes.cpp

```diff
@@ -332,6 +332,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	const Color font_color = mono_color.linear_interpolate(base_color, 0.25);
 	const Color font_color_hl = mono_color.linear_interpolate(base_color, 0.15);
 	const Color font_color_disabled = Color(mono_color.r, mono_color.g, mono_color.b, 0.3);
+	const Color font_color_selection = Color::html("#7d7d7d");
```

Comment:
how this hardcoded is related PR stands for?

Reply:
the color code is kept from the default theme.